dapps/tests/contracts/test/another_storage_spec.js

```diff
@@ -13,8 +13,11 @@ config({
       "SimpleStorage": {
         args: [100]
       },
+      // "AnotherStorage": {
+      //   args: ["$SimpleStorage", "embark.eth"]
+      // },
```

Comment:
If this is for the same of keeping it around, this should probably be removed. Version control's got out back!